Services/UserService/Validators/ErrorsMessages.resx

```diff
@@ -0,0 +1,147 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are we have to have this handling? Why do we use it with resources?

Reply:
I use it the way it’s easier to replace it with another language. And in the tests, I don't need to hardcode the values